examples/with-docker-multi-env/docker/staging/Dockerfile

```
@@ -21,14 +21,14 @@ WORKDIR /app
 COPY --from=deps /app/node_modules ./node_modules
 COPY . .
 # This will do the trick, use the corresponding env file for each environment.
-COPY .env.staging.sample .env.production
+COPY .env.staging.sample .env.staging
 RUN yarn build
 
 # 3. Production image, copy all the files and run next
 FROM node:16-alpine AS runner
 WORKDIR /app
 
-ENV NODE_ENV=production
+ENV NODE_ENV=staging
```

```suggestion
ENV NODE_ENV=production
```
this should still be `production` as `staging` isn't a valid value here and will cause a warning in Next.js